backend/app/src/main/java/com/moh/phlat/backend/service/ProcessDataService.java

```diff
@@ -1,14 +1,16 @@
 package com.moh.phlat.backend.service;
 
+import java.util.Arrays;
+import java.util.List;
+
 import com.moh.phlat.backend.model.ProcessData;
 import com.moh.phlat.backend.model.ProcessDataFilterParams;
 import com.moh.phlat.backend.service.dto.ReportSummary;
 
-import java.util.Arrays;
-import java.util.List;
-
 public interface ProcessDataService {
-	
+	String PHLAT_ERROR_TYPE = "ERROR";
```

Comment:
I guess that there are more types coming once we start processing response from PLR. The right place for the type and error codes should be a separate interface on its own for example MessageInfo?